src/SourceGenerators/Uno.UI.SourceGenerators/Helpers/HashBuilder.cs

```diff
@@ -1,22 +1,27 @@
 ﻿using System;
-using System.Collections.Generic;
-using System.Globalization;
-using System.IO;
 using System.Linq;
+using System.Runtime.InteropServices;
 using System.Security.Cryptography;
 using System.Text;
-using System.Threading.Tasks;
 using Microsoft.CodeAnalysis;
 
 namespace Uno.UI.SourceGenerators.Helpers
 {
 	internal class HashBuilder
 	{
+		private static uint[] _Lookup32 = Enumerable.Range(0, 256).Select(i =>
+		{
+			string s = i.ToString("x2");
+			return ((uint)s[0]) + ((uint)s[1] << 16);
```

Comment:
This could be doable without the string allocation, but probably isn't worth the effort given it's a one-time initialization thing anyway.